sklearn/kernel_ridge/kernel_ridge.py

```diff
@@ -181,5 +179,12 @@ def predict(self, X):
             Returns predicted values.
         """
         check_is_fitted(self, ["X_fit_", "dual_coef_"])
+        # Input validation
         K = self._get_kernel(X, self.X_fit_)
-        return np.dot(K, self.dual_coef_)
+        y = np.dot(K, self.dual_coef_)
+
+        if self.regression is False:  # Multilabel classification
+            # Decoding
+            y = self.label_encoder_.inverse_transform(y)
```

Comment:
Don't we need to do an argmax before this? How is this not failing tests? Am I missing something?

Reply:
As far as I tested with `LabelBinarized`, that is its work. Before you tell me about that class, I had to do that work, but now `LabelBinarized` does the job